src/tests/mocks/handlers.ts

```diff
@@ -0,0 +1,92 @@
+import { rest } from 'msw';
+
+const API_BASE_URL = 'http://localhost:3001';
+
+//Mock data
```

Comment:
🚀 nice:  don't know if this actually mocks your API but good job setting up msw!

Reply:
thank ya thank ya